src/styles.js

```diff
@@ -62,6 +110,7 @@ module.exports = (theme) => ({
           position: 'absolute',
           fontWeight: '400',
           color: theme('colors.gray.500', defaultTheme.colors.gray[500]),
+          left: '0',
```

Comment:
'ol > li::before'.left can be moved to the first item of the DEFAULT variant & removed from the rest of the variant declarations because it is never actually overridden anywhere in the code